src/resources/lv_img_conv.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+import argparse
+import pathlib
+import sys
+import decimal
+from PIL import Image
+
+
+def classify_pixel(value, bits):
+    def round_half_up(v):
+        """python3 implements "propper" "banker's rounding" by rounding to the nearest
+        even number. Javascript rounds to the nearest integer.
+        To have the same output as the original JavaScript implementation add a custom
+        rounding function, which does "school" rounding (to the nearest integer).
+
+        see: https://stackoverflow.com/questions/43851273/how-to-round-float-0-5-up-to-1-0-while-still-rounding-0-45-to-0-0-as-the-usual
+        """
+        return int(decimal.Decimal(v).quantize(decimal.Decimal('1'), rounding=decimal.ROUND_HALF_UP))
```

Comment:
Is there any advantage to using the JS rounding method rather than round-to-even? If it doesn't matter I think we may as well avoid having the extra code
It looks like GIMP agrees with round-to-even as well (if I'm reading your previous PR comments right), and I trust GIMP above lv_image_conv on correctness

Reply:
the GIMP case is for how the PNG is read into memory, the rounding is for how the pixels are rounded before written out. I'd like to keep to be as close as possible to the original here (so I'd like to keep the rounding)